auto3dseg/algorithm_templates/swinunetr/scripts/dummy_runner.py

```diff
@@ -64,15 +65,25 @@ def __init__(self, output_path, data_stats_file, device_id: int = 0):
         with open(data_stats_file) as f_data_stat:
             data_stat = yaml.full_load(f_data_stat)
 
-        pixdim = parser.get_parsed_content("transforms_train#transforms#4#pixdim")
+        pixdim = parser.get_parsed_content("transforms_train#transforms#3#pixdim")
         pixdim = [np.abs(pixdim[_i]) for _i in range(3)]
 
-        if "sizemm" not in data_stat["stats_summary"]["image_stats"]:
-            raise ValueError("The data stats file is generated by older version of MONAI. "
-                             "Please update MONAI >= 1.2 and re-run the data analyzer on your dataset.")
-
-        image_size_mm = data_stat["stats_summary"]["image_stats"]["sizemm"]["percentile_99_5"]
-        self.max_shape = [int(np.ceil(image_size_mm[_l] / pixdim[_l])) for _l in range(3)]
```

Comment:
these are useful workarounds, please be careful when merging upstream into your fork when creating this PR.

Reply:
I added those back